Tweeter/Controllers/TweetController.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+using Tweeter.DAL;
+using Tweeter.Models;
+
+namespace Tweeter.Controllers
+{
+    public class TweetController : ApiController
+    {
+        // GET: api/Tweet
+        public IEnumerable<Tweet> Get()
+        {
+            TweeterRepository repo = new TweeterRepository();
+            List<Tweet> all_tweets = repo.GetAllTweets();
+            return all_tweets;
+        }
+
+      
+
+        // POST: api/Tweet
+        public void Post([FromBody]Tweet tweet)
+        {
+            TweeterRepository repo = new TweeterRepository();
```

Comment:
You can instantiate just one `repo` as a private data member on the controller class rather than instantiating a new one every time we hit one of these controller methods